docs/source/tutorial/kernel.rst

```diff
@@ -370,3 +372,69 @@ You can specify the kernel name by using the ``kernel_name`` keyword argument as
 
 .. note::
    Currently, :func:`cupy.fuse` can fuse only simple elementwise and reduction operations.  Most other routines (e.g. :func:`cupy.matmul`, :func:`cupy.reshape`) are not supported.
+
+.. _jit_kernel_definition:
+
+JIT kernel definition
+---------------------
+
+The :class:`~cupyx.jit.rawkernel` decorator can define raw CUDA kernel from Python function.
+
+In this section, the Python function wrapped with the decorator is called as *target function*.
+
+The target function consists of the elemental scalar operations, and the users have to manage how parallelize them. CuPy's array operations which automatically parallelize the operation (e.g., :func:`cupy.add`, :func:`cupy.sum`) is not supported. If the custom kernel from the array functions are desired, please refer the :ref:`kernel_fusion` section.
+
+Basic Usage
+^^^^^^^^^^^
+
+:class:`cupyx.jit.rawkernel` decorator can define a CUDA global function.
+
+.. doctest::
+
+   >>> @jit.rawkernel()
+   ... def elementwise_copy(x, y, size):
+   ...     tid = jit.blockIdx.x * jit.blockDim.x + jit.threadIdx.x
+   ...     ntid = jit.gridDim.x * jit.blockDim.x
+   ...     for i in range(tid, size, ntid):
+   ...         y[i] = x[i]
+
+   >>> size = cupy.uint32(2 ** 22)
+   >>> x = cupy.random.normal(size=(size,), dtype=cupy.float32)
+   >>> y = cupy.empty((size,), dtype=cupy.float32)
+
+Two kinds of interfaces  to launch the kenrel are supported. See the document of :class:`cupyx.jit.CudaFunction` for details.
```

Comment:
```suggestion
Two kinds of interfaces to launch the kernel are supported, see the documentation of :class:`cupyx.jit.CudaFunction` for details.
```
btw, is any of the interfaces experimental? If so we probably want to mention it here.